static/app/views/onboarding/targetedOnboarding/types.ts

```diff
@@ -0,0 +1,23 @@
+import {PlatformKey} from 'sentry/data/platformCategories';
+import {Organization, Project} from 'sentry/types';
+
+export type StepData = {
+  platform?: PlatformKey | null;
+};
+
+export type StepProps = {
+  active: boolean;
+  onComplete: () => void;
+  orgId: string;
+  organization: Organization;
+  search: string;
+};
+
+export type StepDescriptor = {
+  Component: React.ComponentType<StepProps>;
+  id: string;
+  title: string;
+  FooterComponent?: React.ComponentType<{organization: Organization; project?: Project}>;
```

Comment:
Is this `FooterComponent` used anywhere?
Also, between `FooterComponent` and `hasFooter` you probably only need one.